import-export-cli/utils/utils.go

```diff
@@ -34,222 +34,85 @@ import (
 )
 
 func InvokePOSTRequest(url string, headers map[string]string, body interface{}) (*resty.Response, error) {
-	if Insecure {
-		resty.SetTLSClientConfig(
-			&tls.Config{InsecureSkipVerify: true, // To bypass errors in SSL certificates
-				Renegotiation: TLSRenegotiationMode})
-	} else {
-		resty.SetTLSClientConfig(GetTlsConfigWithCertificate())
-	}
-	if os.Getenv("HTTP_PROXY") != "" {
-		resty.SetProxy(os.Getenv("HTTP_PROXY"))
-	} else if os.Getenv("HTTPS_PROXY") != "" {
-		resty.SetProxy(os.Getenv("HTTPS_PROXY"))
-	} else if os.Getenv("http_proxy") != "" {
-		resty.SetProxy(os.Getenv("http_proxy"))
-	} else if os.Getenv("https_proxy") != "" {
-		resty.SetProxy(os.Getenv("https_proxy"))
-	}
-	resty.SetTimeout(time.Duration(HttpRequestTimeout) * time.Millisecond)
-	resp, err := resty.R().SetHeaders(headers).SetBody(body).Post(url)
```

Comment:
Any reason for changing the common utils code?

Reply:
Since the resty.New() doesn't exist inside the code block, I though it would be alright to move that section out, in order to avoid repetition, and make the implementation inside each method more visible. This helped me easily figure out what are the existing implementations and only add if it doesn't exist. For insecure flag it worked properly, so hope it works for the rest. Do you think it would have other complications, and want me to put everything back? (Please don't say yes)

Reply:
No, the reason for asking this is that we have already done changes to this part. But that's fine, +1 for the changes.